### ISSUE TITLE
Block does not display correctly

### ISSUE THREAD

**Issue by narongporn** [Shorts1999/ptx-smartmatrix#3]
I have install this extension.
when I clicked at SmartMatrix, all command blocks should be listed correctly.
I can't drag any block to programing area. I have to switch to javascript mode in order to use the SmartMatrix Command .
![smartMatrix](https://user-images.githubusercontent.com/43835385/100248582-3c545580-2f6e-11eb-9f57-4558b7221c2e.png)

Moreover, pls provide sample to demonstrate draw bitmap command. It is interesting

**Comment by Shorts1999** [comment]
Hello, thank you for the update! I had not experienced this issue before but maybe it happened with the "recent" update? (I have not updated this in a while).

I will look into it soon!

**Comment by Shorts1999** [comment]
The blocks not correctly displaying has now been resolved!
I've also moved the drawBitmap to the advanced section.

For a further description on how to use it I would suggest looking at some information on the internet, it works just like how many similar Arduino libraries draw it (Like the Adafruit NeoMatrix/GFX): https://learn.adafruit.com/adafruit-gfx-graphics-library/graphics-primitives#bitmaps-2002806-32

Basically you put in a list/array of bytes, every "1" will draw a pixel to the selected colour, a "0" will be ignored.
the width and height have to be put in the function so it knows where to start a new line and the x/y values are a offset of where to place it on the screen.

Hope this somewhat helps, the best way to learn is to just try it out ;)
